base/operators.jl

```diff
@@ -9,11 +9,18 @@ typealias Indices{N} NTuple{N,AbstractUnitRange}
 """
     <:(T1, T2)
 
-Subtype operator, equivalent to `issubtype(T1,T2)`.
+Subtype operator, equivalent to `issubtype(T1, T2)`.
 """
 const (<:) = issubtype
 
 """
+    >:(T1, T2)
+
+Supertype operator, equivalent to `issubtype(T2, T1)`.
+"""
+const (>:)(a::ANY, b::ANY) = issubtype(b, a)
```

Comment:
I believe this needs special inference support to be able to infer `Const(true)` or `Const(false)`.